sccp.c

```diff
@@ -88,6 +88,21 @@ static struct ast_channel *channel_tech_requester(const char *type, struct ast_f
 		break;
 	}
 
+	if (!line) {
+		return NULL;
+	}
+
+	struct sccp_device *device = sccp_line_device(line);
```

Comment:
I don't think its the right place to do that here. It's still too early in the process to start changing the device's state at this point